.sops.yaml

```diff
@@ -74,7 +74,7 @@ creation_rules:
           - *zimbatm
           - *zowoq
           - *adisbladis
-  - path_regex: modules/darwin/hercules-ci/.+\.yaml$
+  - path_regex: modules/darwin/.+\.yaml$
```

Comment:
Time to add sops-nix support for darwin :) ? So far we only support home-manager on this target.

Reply:
> sops-nix support for darwin

Are there any technical issues that need to overcome for that or would it just be a case of adapting the darwin home-manager support for nix-darwin?

Reply:
I would expect it's more a case of adapting it based on what we do for home-manager.